### DISCUSSION TITLE
Links to PDFs and Jupyter Notebooks

### DISCUSSION THREAD

**Discussion by cw-tan** [alshedivat/al-folio#403] [Q&A] [answered]
Hi all, thanks for this amazing template! I'm very inexperienced when it comes to these things so I was hoping if anybody had any advice/suggestions about how I could put a link in the projects/teaching page that will open a PDF in a web browser. Also, is there an easy way to have a link to a Jupyter Notebook rendered in a static website (so it's not interactive, but has all the results of a Jupyter Notebook that's been ran in a .ipynb file), and also have a link for people to download the .ipynb file? I was just wondering if there was a simple way to do this in al-folio. Thank you!

**Comment by GabrielEdogawa** [comment]
Does anyone know the answer? I am still wondering how to insert PDF to this. I saw somebody has done this but it involves intensive HTML coding and modification, which I am not familiar with. It would be fantastic if someone could give a sample code for that. Thanks!

**Comment by seongsukim-ml** [comment]
I found the simple solution!

If you upload the pdf in the `assets/pdf/{name}.pdf `, then the link of the pdf is automatically generated by `https://{blog-name}github.io/assets/pdf/{name}.pdf`. You may directly redirect the page to this link to upload the pdf on the project page.

**Reply by seongsukim-ml** [reply, marked as answer]
Or alternativle, you can use the permalink option like this

```
---
layout: page
title: title
description: a project that redirects to another website
#redirect: https://{blogname}.github.io/assets/pdf/{pdfname}.pdf
permalink: assets/pdf/{pdfname}.pdf
importance: 1
category: study
---
```

**Reply by george-gca** [reply]
The permalink solution was recently introduced in #2863.
